WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -854,6 +862,59 @@ private extension ProductsViewController {
             }
         }
     }
+
+    func checkBlazeBannerVisibility() {
+        viewModel.$shouldShowBlazeBanner
+            .removeDuplicates()
+            .sink { [weak self] shouldShow in
+                guard let self else { return }
+                guard !self.hasErrorLoadingData else {
+                    return // ignores Blaze banner if has error fetching data.
+                }
```

Comment:
❓ If the error banner should be shown while the Blaze banner is already shown, do we want to hide the Blaze banner? or is it already handled when showing the error banner?

Reply:
The [implementation](https://github.com/woocommerce/woocommerce-ios/blob/fa31827aa7a15e345804e29e2af8ec08437aba84/WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift#L768-L770) of the error banner lets the banner replace the header entirely so it's not necessary to hide the banner. But I agree that this is more of an implementation detail, so I've added the call to hide the Blaze banner in case we change something in implementing the error banner.